h/views/api_groups.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid import security
+from pyramid.httpexceptions import HTTPNoContent
+
+from h.views.api import api_config
+
+
+@api_config(route_name='api.group_member',
+            request_method='DELETE',
+            link_name='group.member.remove',
```

Comment:
`group.member.delete` might be more consistent with what we already have? Or do you think that it could be misunderstood that it deletes the user as well?